applications/FluidDynamicsApplication/custom_elements/d_vms_dem_coupled.h

```diff
@@ -0,0 +1,394 @@
+    typedef Vector VectorType;
+
+    /// Matrix type for local contributions to the linear system
+    typedef Matrix MatrixType;
+
+    typedef std::size_t IndexType;
+
+    typedef std::size_t SizeType;
+
+    typedef std::vector<std::size_t> EquationIdVectorType;
+
+    typedef std::vector< Dof<double>::Pointer > DofsVectorType;
+
+    typedef PointerVectorSet<Dof<double>, IndexedObject> DofsArrayType;
+
+    /// Type for shape function values container
+    typedef Kratos::Vector ShapeFunctionsType;
+
+    /// Type for a matrix containing the shape function gradients
+    typedef Kratos::Matrix ShapeFunctionDerivativesType;
+
+    /// Type for an array of shape function gradient matrices
+    typedef GeometryType::ShapeFunctionsGradientsType ShapeFunctionDerivativesArrayType;
+
+    constexpr static unsigned int Dim = DVMS<TElementData>::Dim;
+    constexpr static unsigned int NumNodes = DVMS<TElementData>::NumNodes;
+    constexpr static unsigned int BlockSize = DVMS<TElementData>::BlockSize;
+    constexpr static unsigned int LocalSize = DVMS<TElementData>::LocalSize;
+    constexpr static unsigned int StrainSize = DVMS<TElementData>::StrainSize;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    //Constructors.
+
+    /// Default constuctor.
+    /**
+     * @param NewId Index number of the new element (optional)
+     */
+    DVMSDEMCoupled(IndexType NewId = 0);
+
+    /// Constructor using an array of nodes.
+    /**
+     * @param NewId Index of the new element
+     * @param ThisNodes An array containing the nodes of the new element
+     */
+    DVMSDEMCoupled(IndexType NewId, const NodesArrayType& ThisNodes);
+
+    /// Constructor using a geometry object.
+    /**
+     * @param NewId Index of the new element
+     * @param pGeometry Pointer to a geometry object
+     */
+    DVMSDEMCoupled(IndexType NewId, GeometryType::Pointer pGeometry);
+
+    /// Constuctor using geometry and properties.
+    /**
+     * @param NewId Index of the new element
+     * @param pGeometry Pointer to a geometry object
+     * @param pProperties Pointer to the element's properties
+     */
+    DVMSDEMCoupled(IndexType NewId, GeometryType::Pointer pGeometry, Properties::Pointer pProperties);
+
+    /// Destructor.
+    ~DVMSDEMCoupled() override;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+
+    /// Create a new element of this type
+    /**
+     * Returns a pointer to a new DVMSDEMCoupled element, created using given input
+     * @param NewId the ID of the new element
+     * @param ThisNodes the nodes of the new element
+     * @param pProperties the properties assigned to the new element
+     * @return a Pointer to the new element
+     */
+    Element::Pointer Create(IndexType NewId,
+                            NodesArrayType const& ThisNodes,
+                            Properties::Pointer pProperties) const override;
```

Comment:
```suggestion
    Element::Pointer Create(
        IndexType NewId,
        NodesArrayType const& ThisNodes,
        Properties::Pointer pProperties) const override;
```
(minor style guide issue, please do so for the rest of methods).

Reply:
Ok, I thought  I had written rightly